perfkitbenchmarker/providers/rackspace/rackspace_disk.py

```diff
@@ -15,192 +15,263 @@
 
 Volumes can be created, deleted, *attached to VMs, and *detached from VMs. Also
```

Comment:
Possible extra "*"s here before "attached" and "detached". Did you write it as a bulleted list and then reformat as a paragraph?